doc/rippled-example.cfg

```diff
@@ -842,7 +842,7 @@ port_wss_admin
 port = 5005
 ip = 127.0.0.1
 admin = 127.0.0.1
-protocol = https
+protocol = http
```

Comment:
Yes it does and I had tried that configuration but unfortunately it seems to choose https.